### PR TITLE
[16.0][ADD] purchase_order_downpayment

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

File: purchase_order_downpayment/wizard/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import purchase_order_downpayment_wiz
```
Comment (legalsylvain) [inline purchase_order_downpayment/wizard/__init__.py]:
```suggestion
from . import purchase_order_downpayment_wizard
```

ususally in OCA, use the name of the model as the name of the file.

File: purchase_order_downpayment/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from . import payment
+from . import purchase
```
Comment (legalsylvain) [inline purchase_order_downpayment/models/__init__.py]:
```suggestion
from . import account_payment
from . import purchase_order
```

File: purchase_order_downpayment/wizard/purchase_order_downpayment_wiz_view.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<odoo>
+    <record id="view_po_downpayment_wizard" model="ir.ui.view">
+        <field name="name">Ppurchase order down payment</field>
```
Comment (legalsylvain) [inline purchase_order_downpayment/wizard/purchase_order_downpayment_wiz_view.xml]:
```suggestion
        <field name="name">Purchase order down payment</field>
```

**Review by legalsylvain — COMMENTED** [review]
thanks for sharing your module.

Partial review / some minork remarks inline.

**Comment by yostashiro** [comment]
What scenarios does this module address that `purchase_advance_payment` and `purchase_deposit` do not?

**Comment by SMaciasOSI** [comment]
> What scenarios does this module address that `purchase_advance_payment` and `purchase_deposit` do not?

This module allows the buyer to request a payment to the accounting team.

`purchase_advance_payment` creates a bill.
`purchase_deposit` creates a payment with Posted status and does not link the payment to the PO's record

**Review by max3903 — APPROVED** [review]

**Review by SMaciasOSI — APPROVED** [review]

**Comment by legalsylvain** [comment]
hi @max3903. sorry no time to review deeply. but this PR is approved and can be merged without my review. 
thanks for the contribution !

**Pull request merged**